core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -433,7 +433,9 @@ trait SubmissionComponent {
                new DateTime(wr.statusLastChangedDate.getTime),
                entityRef,
                workflowResolutions.sortBy(_.inputName), // enforce consistent sorting
-               messages
+               messages,
+               wr.cost.map(_.floatValue),
+               wr.cost.map(_ => WorkflowCostTypes.Estimated)
```

Comment:
when retrieving a workflow from the db, if that workflow has a value for `cost`, the value is an estimated value.